From e58c13128295411e6f15ebe3e3d1746ebf37840c Mon Sep 17 00:00:00 2001
From: Michele Locati <michele@locati.it>
Date: Wed, 21 Nov 2018 14:38:53 +0100
Subject: [PATCH] Define missing potentially defines in sspi_errmsg.c

---
 plugins/auth/sspi_errmsg.c | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/plugins/auth/sspi_errmsg.c b/plugins/auth/sspi_errmsg.c
index 558bedcb..336091ca 100644
--- a/plugins/auth/sspi_errmsg.c
+++ b/plugins/auth/sspi_errmsg.c
@@ -29,6 +29,30 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <windows.h>
 #include <stdio.h>
 
+#ifndef SEC_E_INVALID_PARAMETER
+#define SEC_E_INVALID_PARAMETER _HRESULT_TYPEDEF_(0x8009035D)
+#endif
+
+#ifndef SEC_E_DELEGATION_POLICY
+#define SEC_E_DELEGATION_POLICY _HRESULT_TYPEDEF_(0x8009035E)
+#endif
+
+#ifndef SEC_E_POLICY_NLTM_ONLY
+#define SEC_E_POLICY_NLTM_ONLY _HRESULT_TYPEDEF_(0x8009035F)
+#endif
+
+#ifndef SEC_E_NO_CONTEXT
+#define SEC_E_NO_CONTEXT _HRESULT_TYPEDEF_(0x80090361)
+#endif
+
+#ifndef SEC_E_PKU2U_CERT_FAILURE
+#define SEC_E_PKU2U_CERT_FAILURE _HRESULT_TYPEDEF_(0x80090362)
+#endif
+
+#ifndef SEC_E_MUTUAL_AUTH_FAILED
+#define SEC_E_MUTUAL_AUTH_FAILED _HRESULT_TYPEDEF_(0x80090363)
+#endif
+
 #define ERRSYM(x) {x, #x}
 static struct {
   int error;
