From 3a042b65998eb3b05382f081fbf9548d88997487 Mon Sep 17 00:00:00 2001
From: Michele Locati <michele@locati.it>
Date: Wed, 21 Nov 2018 14:45:44 +0100
Subject: [PATCH] Cast constant string as char *

---
 plugins/auth/sspi_client.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/auth/sspi_client.c b/plugins/auth/sspi_client.c
index 75f5a1f5..9d02fce7 100644
--- a/plugins/auth/sspi_client.c
+++ b/plugins/auth/sspi_client.c
@@ -79,7 +79,7 @@ int auth_client(char *principal_name, char *mech, MYSQL *mysql, MYSQL_PLUGIN_VIO
 
   if (!mech || strcmp(mech, "Negotiate") != 0)
   {
-    mech= "Kerberos";
+    mech= (char *)"Kerberos";
   }
 
   sspi_err = AcquireCredentialsHandle(
