From 47efae23bbd63b842062aabb823f1b322049f2bd Mon Sep 17 00:00:00 2001
From: Michele Locati <michele@locati.it>
Date: Wed, 21 Nov 2018 14:49:23 +0100
Subject: [PATCH] Remove unused ParamSize variable in caching_sha2_pw.c

---
 plugins/auth/caching_sha2_pw.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/plugins/auth/caching_sha2_pw.c b/plugins/auth/caching_sha2_pw.c
index a76ca8e1..b08e9423 100644
--- a/plugins/auth/caching_sha2_pw.c
+++ b/plugins/auth/caching_sha2_pw.c
@@ -255,7 +255,6 @@ static int auth_caching_sha2_client(MYSQL_PLUGIN_VIO *vio, MYSQL *mysql)
   LPBYTE der_buffer= NULL;
   DWORD der_buffer_len= 0;
   CERT_PUBLIC_KEY_INFO *publicKeyInfo= NULL;
-  DWORD ParamSize= sizeof(DWORD);
   int publicKeyInfoLen;
 #endif
 
