From 17fb839e1c2e746a123eafec13548b53890f89d1 Mon Sep 17 00:00:00 2001
From: Michele Locati <michele@locati.it>
Date: Wed, 21 Nov 2018 15:05:46 +0100
Subject: [PATCH] Add bcrypt to the libraries for caching_sha2_password

---
 plugins/auth/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/plugins/auth/CMakeLists.txt b/plugins/auth/CMakeLists.txt
index 9d284219..339afa5d 100644
--- a/plugins/auth/CMakeLists.txt
+++ b/plugins/auth/CMakeLists.txt
@@ -74,6 +74,9 @@ IF(WITH_SSL)
     SET(CRYPT_SOURCE ${CC_SOURCE_DIR}/libmariadb/secure/gnutls_crypt.c)
     SET(CACHING_SHA2_LIBS ${SSL_LIBRARIES})
   ENDIF()
+  IF(WIN32)
+    SET(CACHING_SHA2_LIBS ${CACHING_SHA2_LIBS} bcrypt)
+  ENDIF()
   REGISTER_PLUGIN(TARGET caching_sha2_password
                 TYPE MARIADB_CLIENT_PLUGIN_AUTH
                 CONFIGURATIONS DYNAMIC STATIC OFF
