From 1a579bbf05410e9cb7eab3f3c1d074945182c02a Mon Sep 17 00:00:00 2001
From: Michele Locati <michele@locati.it>
Date: Wed, 21 Nov 2018 15:08:48 +0100
Subject: [PATCH] Fix type of pcbStructInfo param of CryptDecodeObjectEx

---
 plugins/auth/caching_sha2_pw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/auth/caching_sha2_pw.c b/plugins/auth/caching_sha2_pw.c
index a76ca8e1..9719fbf2 100644
--- a/plugins/auth/caching_sha2_pw.c
+++ b/plugins/auth/caching_sha2_pw.c
@@ -259,4 +259,4 @@ static int auth_caching_sha2_client(MYSQL_PLUGIN_VIO *vio, MYSQL *mysql)
-  int publicKeyInfoLen;
+  DWORD publicKeyInfoLen;
 #endif
 
   /* read error */
